bower.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "estraverse",
+  "main": "dist/estraverse.js",
+  "version": "4.1.0",
```

Comment:
No need to add `version` field. https://github.com/bower/spec/blob/master/json.md#version

> Use git or svn tags instead. This field is ignored by Bower.